### PR TITLE
Update Minimalistic 'Jetpack compose rework' from my 'jetpack-compose-rework'

### PR DESCRIPTION
**Pull request by hafizsaadiftikhar**

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -23,5 +23,5 @@ dependencyResolutionManagement {
         mavenCentral()
     }
 }
-rootProject.name = "Price Converter"
+rootProject.name = "Minimalistic Apps"
```
Comment (hank-milliken) [inline settings.gradle:26]: Why this change? Project is called "Price Converter"

**Comment by hank-milliken** [comment]
There are commited files in `.gradle/*` directory. I think they are there by mistake. What is the reason for them in repository? If it is mistake, please remove them from PR.

File: settings.gradle (hunk 1 above)
Comment (hafizsaadiftikhar) [reply settings.gradle:26]: I changed it because you send the link to the color scheme and you mentioned the name as Minimalistic app that's why I changed it.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name">Price Converter</string>
+    <string name="app_name">Minimalistic Apps</string>
```
[the remark this answers is not in the document]
Comment (hafizsaadiftikhar) [reply app/src/main/res/values/strings.xml:2]: I'll change it Price Converter

**Comment by hafizsaadiftikhar** [comment]
For the .gradle/* It was disturbed resolving conflicts
